app/vendor/ToolsAPI.js

```diff
@@ -279,7 +279,7 @@ sap.ui.define(['jquery.sap.global', 'sap/ui/core/library', 'sap/ui/Global', 'sap
                 if (bindingContextModel) {
                     model.mode = bindingContextModel.getDefaultBindingMode();
                     model.type = bindingContextModel.getMetadata().getName();
-                    model.data = bindingContextModel.getData ? bindingContextModel.getData() : undefined;
+                    model.data = bindingContextModel.getData ? bindingContextModel.getData('/') : undefined;
```

Comment:
`getData` is deprecated for the ODataModel and won't return the expected data. I just talked to @goligo and it should be a better solution to use the internal `bindingContextModel.oData` property.